Makefile

```diff
@@ -0,0 +1,73 @@
+# Copyright 2021 The kube-storage Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SHELL:=/bin/bash
+
+PROTOC_VERSION := 3.14.0
+PROTOC_GEN_GO_VERSION := 1.25.0
+PROTOC_GEN_GO_GRPC_VERSION := 1.1.0
+
+PROTOC_FOUND := $(shell ./bin/protoc --version 2> /dev/null)
+PROTOC_GEN_GO_FOUND := $(shell ./bin/protoc-gen-go --version 2> /dev/null)
+PROTOC_GEN_GO_GRPC_FOUND := $(shell ./bin/protoc-gen-go-grpc --version 2> /dev/null)
+
+
+ifeq ("$(PROTOC_FOUND)","libprotoc ${PROTOC_VERSION}")
```

Comment:
have you tested this? there is a mix of `$(...)` (make) and `${...}` (bash) notation

Reply:
Yes, I did both are working similarly for me. Anyway, I have changed it to curly.